src/Be.Vlaanderen.Basisregisters.Projector/Internal/ConnectedProjectionContext.cs

```diff
@@ -100,6 +113,20 @@ internal class ConnectedProjectionContext<TContext> : IConnectedProjectionContex
                 _envelopeFactory.Create(message),
                 cancellationToken);
 
+        public async Task SetErrorMessage(
+            ConnectedProjectionName projectionName,
+            Exception exception,
+            CancellationToken cancellationToken)
+        {
+            var message = exception.Message + "Stacktrace: " + exception.StackTrace;
```

Comment:
Do you need a space before `Stacktrace: `?

Could also do String.Concat, or string join with a space as separator :)

Reply:
Also, check if `exception.ToString()` doesn't already give you a nicer format than getting `Message` and `StackTrace` yourself.